backend/logic/users/users.js

```diff
@@ -167,10 +175,47 @@ var getCompletedCourses = async studentID => {
   return courses;
 };
 
+var getStudentData = async studentID => {
+  let data;
+
+  let conn = await mysql.getNewConnection();
+
+  let courses, major, minor;
+  try {
+    courses = await conn.query(
+      `SELECT course_code, semester
+    FROM course_offerings
```

Comment:
picky, but formatting is off here

Reply:
what part of it is off?

Reply:
from should be indented more i think

Reply:
Not according to prettier plugin it seems. First line is indented but the subsequent ones aren't (like an indent at the start of a paragraph)

Reply:
oh.. ew